docs/source/release/v6.2.0/muon.rst

```diff
@@ -71,28 +68,41 @@ Improvements
 - The plotting has been updated for better stability.
 - The plotting now has autoscale active by default.
 - It is now possible to load nexusV2 files in the GUI.
-- Added a table to store phasequads in the phase tab, phasequads also no longer automatically delete themselves
+- Added a table to store phasequads in the phase tab. Phasequads also no longer automatically delete themselves
```

Comment:
Can we rephrase the last sentence to 'Also, phasequads no longer delete themselves automatically.'